src/django/api/tests.py

```diff
@@ -148,6 +170,22 @@ def setUpTestData(cls):
             approved_by=cls.validator,
         )
 
+        approved = Submission.objects.create(
```

Comment:
IMO we shouldn't overwrite the `approved` variable name, which can get confusing! (I see this was already happening with some names here.)

Reply:
Whoops, good catch!